taskcluster/kinds/finetune-student/kind.yml

```
@@ -104,7 +104,7 @@ tasks:
                     finetune
                     {src_locale}
                     {trg_locale}
-                    $MOZ_FETCHES_DIR/corpus
+                    $MOZ_FETCHES_DIR/corpus.moses
```

From my idea in another comment, `tok-*` would look like this here, which to me reads a bit more intuitively.

```suggestion
                    $MOZ_FETCHES_DIR/corpus.tok-moses
```